sound/soc/intel/boards/sof_ssp_common.c

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-2.0-only
+//
+// Copyright(c) 2023 Intel Corporation. All rights reserved.
+
+#include <linux/device.h>
+#include <sound/soc-acpi.h>
+#include "sof_cirrus_common.h"
+#include "sof_maxim_common.h"
+#include "sof_realtek_common.h"
```

Comment:
do we really need to include all these vendor-specific headers? That sounds like a partitioning of code that's not quite right?

Reply:
The ACPI HID is defined in sof_<vendor>_common.h so those headers are included here.

Reply:
comment is still valid, I think you just need to include sof_vendor_common.h ?

Reply:
I mean the sof_cirrus_common.h, sof_maxim_common.h, and sof_realtek_common.h. Unless we define the HID definition in sof_ssp_common.h and leave those file untouched.